scripts/Interface/ui/sans_isis/sans_data_processor_window.ui

```diff
@@ -339,7 +339,7 @@ QGroupBox::title {
           <item>
            <widget class="QTabWidget" name="settings_tab_widget">
             <property name="currentIndex">
-             <number>0</number>
+             <number>2</number>
```

Comment:
This changes the default tab of the settings section to be the "Adjustment" tab - is that the intention?